generated/nifgen/enums.py

```
@@ -600,199 +571,196 @@ class StartedEventPulseWidthUnits(Enum):
 
 
 class SynchronizationSource(Enum):
```

This is obsolete. It's for Traditional DAQ devices which are all EOL.
Attribute 1150111 (SYNCHRONIZATION) uses it, should be removed.
Function ConfigureSynchronization also uses it, that one's already not generated.

---

Working on a different PR that will not codegen unreferenced enums

---

> Working on a different PR that will not codegen unreferenced enums

1. silently drop unreferenced enums
2. emit a generation time warning when it runs into this
3. error unless we mark the enum as codegen-method 'no'

I prefer 2 or 3, because they are less error prone. I can see a situation in which we don't add the enum info to a parameter in a function by mistake, and it get silently dropped. 2 and 3 make that more unlikely as it forces us to take a double look.